packages/orchestration/src/cosmos-api.ts

```diff
@@ -54,6 +54,29 @@ export type IBCConnectionInfo = {
   };
 };
 
+/**
+ * https://github.com/cosmos/chain-registry/blob/master/assetlist.schema.json
+ */
+export type CosmosAssetInfo = {
+  base: Denom;
+  name: string;
+  display: string;
+  symbol: string;
+  denom_units: Array<{ denom: Denom; exponent: number }>;
+  traces?: Array<{
+    type: 'ibc';
+    counterparty: {
+      chain_name: string;
+      base_denom: Denom;
+      channel_id: IBCChannelID;
+    };
+    chain: {
+      channel_id: IBCChannelID;
+      path: string;
+    };
+  }>;
```

Comment:
Do we plan to put all of this in vstorage? Seems we're currently only using `base` and `traces`?

Reply:
Copying all of [cosmoshub/assetlist.json](https://github.com/cosmos/chain-registry/blob/master/cosmoshub/assetlist.json) is easier than thinking.

Seems like lots of it is useful for UIs and such. I think I heard @dtribble  say other chains were talking about putting more of this metadata on-chain.

But I don't feel strongly one way or another. It's only a test fixture, so far.